py/tests/commands/test_export_command.py

```diff
@@ -0,0 +1,72 @@
+from configu import InMemoryConfigStore, ConfigSet, ConfigSchema, EvalCommand, ExportCommand
+from unittest import mock
+import os
+
+def test_export_empty_value():
+    store = InMemoryConfigStore()
+    config_set = ConfigSet("")
+    schema = ConfigSchema("tests/staticfiles/config_schemas/valid/empty.cfgu.json")
+    eval_output = EvalCommand(
+        store=store,
+        set=config_set,
+        schema=schema,
+    ).run()
+    export_output = ExportCommand(eval_output).run()
+    assert export_output == {}
+
+@mock.patch.dict(os.environ, {}, clear=True)
+def test_export_value():
+    store = InMemoryConfigStore()
+    config_set = ConfigSet("")
+    schema = ConfigSchema("tests/staticfiles/config_schemas/valid/basic.cfgu.json")
+    eval_output = EvalCommand(
+        store=store,
+        set=config_set,
+        schema=schema,
+        configs={"GREETING": "hello"}
+    ).run()
+    export_output = ExportCommand(eval_output).run()
+    assert export_output == {"GREETING": "hello"}
+
+@mock.patch.dict(os.environ, {}, clear=True)
+def test_export_sets_environment_variables():
+    store = InMemoryConfigStore()
+    config_set = ConfigSet("")
+    schema = ConfigSchema("tests/staticfiles/config_schemas/valid/basic.cfgu.json")
+    eval_output = EvalCommand(
+        store=store,
+        set=config_set,
+        schema=schema,
+        configs={"GREETING": "hello"}
+    ).run()
+    ExportCommand(eval_output).run()
+    assert os.environ.get("GREETING") == "hello"
+
+@mock.patch.dict(os.environ, {"GREETING": "bonjour"}, clear=True)
+def test_export_environment_variables_overrides():
+    store = InMemoryConfigStore()
+    config_set = ConfigSet("")
+    schema = ConfigSchema("tests/staticfiles/config_schemas/valid/basic.cfgu.json")
+    eval_output = EvalCommand(
+        store=store,
+        set=config_set,
+        schema=schema,
+        configs={"GREETING": "hello"}
+    ).run()
+    ExportCommand(eval_output).run()
+    assert os.environ.get("GREETING") == "hello"
+
+#TODO: This test seems to be failing. Is this a bug in the export command or intended behaviour?
```

Comment:
@RonConfigu / @rannn505 Am I right in thinking that the intended behaviour of `override=False` is to only set environment variables if the _don't_ already exist (i.e. not overriding existing env vars)?

Reply:
Correct 💯 
export_command.py line 42 should be:
```python
if override or key not in os.environ:
```
added `not` to the if statement